### PR TITLE
update boba network

### PR DESCRIPTION
**Pull request by alexandriaroberts**
Issue: https://app.asana.com/0/1199528735768693/1202221740846359/f

Screenshot 2022-05-05 at 09 08 03

### CODE REVIEW

**Review by IainM22 — APPROVED** [review]
🔥

**Review by benface — APPROVED** [review]
LGTM! Would it be possible to increase the gap between the last line of networks and the `*In beta` note? It looks a little bit too small to me.

**Comment by alexandriaroberts** [comment]
> LGTM! Would it be possible to increase the gap between the last line of networks and the `*In beta` note? It looks a little bit too small to me.

I have increased the gap and updated font-size as well which was asked before by Carl in here. https://app.asana.com/0/1199528735768693/1201841234561069/f

File: pages/[locale]/index.tsx [hunk 1]
```diff
@@ -409,7 +415,7 @@ const Index: NextPage<IndexProps> = ({ navItems }: IndexProps) => {
             </Flex.Column>
           ))}
         </ul>
-        <Paragraph size="14px">*{t('index.supportedNetworks.betaWarning')}</Paragraph>
+        <Text.P14 sx={{ pt: '16px' }}>*{t('index.supportedNetworks.betaWarning')}</Text.P14>
```
Comment (benface) [inline pages/[locale]/index.tsx:418]: Tiny thing, but I would use margin rather than padding here. You're pushing this whole element down.

**Review by benface — APPROVED** [review]
Thank you for fixing the gap! 👍

File: pages/[locale]/index.tsx (hunk 1 above)
Comment (alexandriaroberts) [reply pages/[locale]/index.tsx:418]: Got it, thank you! Done! :)

**Pull request merged**